pkg/controller/README.md

```diff
@@ -0,0 +1,17 @@
+### Controller local test
+#### Run inside the cluster
+1.run make docker.all to push image with your HUB and TAG
+
+2.update deploy/operator.yaml to point to your image
+
+3.run kubectl apply --recursive deploy/
```

Comment:
this is wrong.  instead use `kubectl apply -k deploy/` followed by an application of the CRD file.

Reply:
md formatting requires a 1 here.